README.md

```diff
@@ -49,3 +49,19 @@ Returns a function for generating random numbers with an [Irwin–Hall distribut
 <a name="randomExponential" href="#randomExponential">#</a> d3.<b>randomExponential</b>(<i>lambda</i>)
 
 Returns a function for generating random numbers with an [exponential distribution](https://en.wikipedia.org/wiki/Exponential_distribution) with the rate *lambda*; equivalent to time between events in a [Poisson process](https://en.wikipedia.org/wiki/Poisson_point_process) with a mean of 1 / *lambda*. For example, exponential(1/40) generates random times between events where, on average, one event occurs every 40 units of time.
+
+<a name="randomPareto" href="#randomPareto">#</a> d3.<b>randomPareto</b>(<i>alpha</i>)
+
+Returns a function for generating random numbers with an [pareto distribution](https://en.wikipedia.org/wiki/Pareto_distribution) with the shape *alpha*.
+
+<a name="randomBernoulli" href="#randomBernoulli">#</a> d3.<b>randomBernoulli</b>(<i>p</i>)
+
+Returns a function for generating either 1 or 0 according to a [Bernoulli distribution](https://en.wikipedia.org/wiki/Binomial_distribution) with 1 being returned with success probability *p* and 0 with failure probability *q* = 1 - *p*.
+
+<a name="randomGeometric" href="#randomGeometric">#</a> d3.<b>randomGeometric</b>(<i>p</i>)
+
+Returns a function for generating numbers with a [Geometric distribution](https://en.wikipedia.org/wiki/Geometric_distribution) with success probability *p*.
+
+<a name="randomBinomial" href="#randomBinomial">#</a> d3.<b>randomBinomial</b>(<i>n</i>, <i>p</i>)
+
+Returns a function for generating random numbers with a [Binomial distribution](https://en.wikipedia.org/wiki/Binomial_distribution) with *n* the number of trials and *p* the probability of success in each trial.
```

Comment:
The README should document the expected ranges of all input arguments (for example if it’s a real number in [0, 1] or a positive integer etc.).